src/main/scala/edu/ie3/powerFactory2psdm/model/RawPfGridModel.scala

```diff
@@ -5,24 +5,23 @@
  */
 
 package edu.ie3.powerFactory2psdm.model
-
 import edu.ie3.powerFactory2psdm.model.RawPfGridModel.{
   ExtGrid,
-  LineTypes,
-  Lines,
-  Loads,
-  LoadsLV,
-  LoadsMV,
-  Nodes,
   PowerPlants,
-  ProjectSettings,
+  Trafos3w,
+  Lines,
   Pvs,
-  StatGen,
   Switches,
-  TrafoTypes2w,
   TrafoTypes3w,
+  LoadsLV,
+  LoadsMV,
+  Nodes,
   Trafos2w,
-  Trafos3w
+  StatGen,
+  Loads,
+  ProjectSettings,
+  LineTypes,
+  TrafoTypes2w
```

Comment:
Pls fix naming issue in line 73.

Reply:
This is not considered by the SchemaGenerator atm. It just writes out attributes as it reads it. I would rather not address that here, as adjusting the SchemaGenerator would take me some time and i might blow that shit up in the process 💣 

Opened an issue (#101)